generated/mocks/generate.go

```diff
@@ -19,5 +19,6 @@
 // THE SOFTWARE.
 
 //go:generate sh -c "mockgen -package=kv -destination=$GOPATH/src/$PACKAGE/kv/mock_store.go -source=$GOPATH/src/$PACKAGE/kv/store.go"
+//go:generate sh -c "mockgen -package=cluster -destination=$GOPATH/src/$PACKAGE/mock_client.go -source=$GOPATH/src/$PACKAGE/client.go"
```

Comment:
Also probably change the package name to `client` if you move `client.go` to its own package